### PR TITLE
Convert build serializable response from JS to TS

### PR DESCRIPTION
**Pull request by wariofdz**
### Changed:

- Transformed file buildSerializableResponse from JS to TS.

### CODE REVIEW

File: packages/frontend/web-ui/src/common/http/helpers/buildSerializableResponse.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Response } from '../../../../../../api/libraries/api-http-client/lib/cjs';
```
Comment (notaphplover) [inline packages/frontend/web-ui/src/common/http/helpers/buildSerializableResponse.ts:1]:
```suggestion
import { Response } from '@cornie-js/api-http-client';
```

File: packages/frontend/web-ui/src/common/http/helpers/buildSerializableResponse.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+export const buildSerializableResponse = (
+  response: Response,
+): { body: Response; statusCode: number } => {
```
Comment (notaphplover) [inline packages/frontend/web-ui/src/common/http/helpers/buildSerializableResponse.ts:5]: body is not a Response

**Review by notaphplover — CHANGES_REQUESTED** [review]
Let's try to figure out the right way to type the function

File: packages/frontend/web-ui/src/common/http/helpers/buildSerializableResponse.ts (hunk 2 above)
Comment (wariofdz) [reply packages/frontend/web-ui/src/common/http/helpers/buildSerializableResponse.ts:5]: I guess it's an object but... How do you know the type of this object or its structure?

**Review by notaphplover — APPROVED** [review]

**Pull request merged**